src/audio/crossover/crossover.c

```diff
@@ -837,8 +837,6 @@ static int crossover_reset(struct comp_dev *dev)
 
 	cd->crossover_process = NULL;
 	cd->crossover_split = NULL;
-	cd->config = NULL;
```

Comment:
You appear to be deleting the same line, I think you might need to rebase

Reply:
The other one is `config_new` which should be resolved in later PR(switch to use new model_handler).

Reply:
im open to merging the PRs but this on its own wont work